components/BlogListing.js

```diff
@@ -0,0 +1,67 @@
+const dotCMSApi = require('../utils/dotcms/dotcmsApi');
+
+import Link from 'next/link';
+import DotcmsImage from './DotcmsImage';
+import Loader from './Loader';
+import useDotCMSApi from '../hooks/useDotCMSApi';
+
+const DateText = ({ value }) => {
+    const monthNames = [
+        'January',
+        'February',
+        'March',
+        'April',
+        'May',
+        'June',
+        'July',
+        'August',
+        'September',
+        'October',
+        'November',
+        'December'
+    ];
+    const d = new Date(value);
+
+    return <time dateTime={value}>{`${monthNames[d.getMonth()]} ${d.getUTCDate()}, ${d.getFullYear()}`}</time>;
+};
+
+const BlogListing = () => {
```

Comment:
not sure if this file, should be moved to a `BLOG` folder or in the `Layout` folder next to the `Footer`, thoughts?

Reply:
I like that idea, BUT we have to make sure we all know this components exists so in case we have to do something similar or reuse we can use it and maybe move it to a proper place.

Reply:
we may need it here
![image](https://user-images.githubusercontent.com/37185433/69435631-896f6d00-0d05-11ea-8bfc-053a8b7172e5.png)

Reply:
A listing components in the detail?

Reply:
@fmontes ooops, that was meant for the time/date utility you created